community/knative/charts/serving/charts/monitoring/charts/prometheus/Chart.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: v1
+appVersion: "0.2.3"
+description: Prometheus Metrics Helm chart for Knative Monitoring
+name: prometheus
+version: 0.1.0
+icon: https://raw.githubusercontent.com/knative/docs/master/images/logo/rgb/knative-logo-rgb.jpg
+kubeVersion: ">=1.10"
+tillerVersion: ">=2.10.0"
```

```suggestion
tillerVersion: ">=2.9.1"
```
Needs to support at least 2.9.1 for ICP 3.1.x support.